serving/installing-logging-metrics-traces.md

```diff
@@ -99,36 +92,38 @@ for request traces.
 
 ![Create logstash-* index](images/kibana-landing-page-configure-index.png)
 
-- To create the second index, select `Create Index Pattern` button on top left
-  of the page. Enter `zipkin*` to `Index pattern` and select `timestamp_millis`
-  from `Time Filter field name` and click on `Create` button.
-
+Visit [Accessing Logs](./accessing-logs.md) to learn more about logs.
 
-## Stackdriver, Prometheus & Grafana Setup
+### Stackdriver
 
-You must configure and build your own Fluentd image if either of the following are true:
+To configure and setup monitoring:
 
- * Your Knative Serving component is not hosted on a Google Cloud Platform (GCP) based cluster.
- * You want to send logs to another GCP project.
+1. Clone the Knative Serving repository:
 
-To configure and setup monitoring:
+    ```shell
+    git clone https://github.com/knative/serving knative-serving
+    cd knative-serving
+    git checkout v0.2.1
+    ```
 
 1. Choose a container image that meets the
    [Fluentd image requirements](fluentd/README.md#requirements). For example, you can use a
    public image. Or you can create a custom one and upload the image to a
    container registry which your cluster has read access to.
-2. Follow the instructions in
+   You must configure and build your own Fluentd image if either of the following are true:
```

Comment:
Do you need to update any of the mentioned YAML files with the URL of the stackdriver image if building your own Fluentd?